src/idl_gen_rust.cpp

```diff
@@ -32,14 +32,19 @@ static std::string GeneratedFileName(const std::string &path,
 // snake_case_indentifier.
 std::string MakeSnakeCase(const std::string &in) {
   std::string s;
-  for (size_t i = 0; i < in.length(); i++) {
-    if (islower(in[i])) {
-      s += static_cast<char>(in[i]);
+  s += static_cast<char>(tolower(in[0]));
+  for (size_t i = 1; i < in.length(); i++) {
+    if (in[i] == '_') {
+      continue;
```

Comment:
Is there a way to make this easier to understand? Using `continue` to skip underscores, then writing them in other branches, is hard to process mentally.